mmdet3d/models/necks/pointnet2_fp_neck.py

```diff
@@ -0,0 +1,66 @@
+from mmcv.runner import BaseModule
+from torch import nn as nn
+
+from mmdet3d.ops import PointFPModule
+from mmdet.models import NECKS
+
+
+@NECKS.register_module()
+class PointNetFPNeck(BaseModule):
```

Comment:
I'm not sure whether we need to follow the structure of [PointNetSSG](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/models/backbones/pointnet2_sa_ssg.py), which incorporates the FP module into itself.

Reply:
@ZwwWayne and I think the FP modules put in necks is better so I create this file.